### PR TITLE
Fix new Sphinx 3.0 stricter warning

### PR DESCRIPTION
**Pull request by shwang**
Sphinx 3.0 caught a weird formatting thing where we ended up generating docs for each NamedTuple property twice. (We listed each field/property in the docs as a property and also an attribute under the old regime)

```
Warning, treated as error:
/venv/lib/python3.7/site-packages/imitation/util/rollout.py:docstring of
imitation.util.rollout.Trajectory.acts:1:duplicate object description of
imitation.util.rollout.Trajectory.acts, other instance in
imitation.util, use :noindex: for one of them
"""

### CODE REVIEW

File: src/imitation/util/rollout.py [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
-
-   Attributes:
-    acts: Actions, shape (trajectory_len, ) + action_shape.
-    obs: Observations, shape (trajectory_len+1, ) + observation_shape.
-    rews: Reward, shape (trajectory_len, ).
-    infos: A list of info dicts, length (trajectory_len, ).
-  """
+  """A trajectory, e.g. a one episode rollout from an expert policy."""
   acts: np.ndarray
+  """Actions, shape(trajectory_len, ) + action_shape."""
```
Comment (AdamGleave) [inline src/imitation/util/rollout.py:21]:
```suggestion
  """Actions, shape (trajectory_len, ) + action_shape."""
```

File: src/imitation/util/rollout.py [hunk 2]
```diff
@@ -16,18 +16,18 @@
...
+
   obs: np.ndarray
+  """Observations, shape(trajectory_len + 1, ) + observation_shape."""
```
Comment (AdamGleave) [inline src/imitation/util/rollout.py:24]:
```suggestion
  """Observations, shape (trajectory_len + 1, ) + observation_shape."""
```

File: src/imitation/util/rollout.py [hunk 3]
```diff
@@ -16,18 +16,18 @@
...
+
   rews: np.ndarray
+  """Reward, shape(trajectory_len, )."""
```
Comment (AdamGleave) [inline src/imitation/util/rollout.py:27]:
```suggestion
  """Reward, shape (trajectory_len, )."""
```

File: src/imitation/util/rollout.py [hunk 4]
```diff
@@ -16,18 +16,18 @@
...
+
   infos: Optional[List[dict]]
+  """A list of info dicts, length(trajectory_len, )."""
```
Comment (AdamGleave) [inline src/imitation/util/rollout.py:30]:
```suggestion
  """A list of info dicts, length trajectory_len."""
```

**Comment by shwang** [comment]
Oh, vim+Pycharm pasting was weird and I didn't notice this. Thanks for adding the auto config.

Giving the docs a quick look now.

**Pull request merged**